python/couler/couler/client.py

```diff
@@ -38,6 +38,8 @@ def run(args):
         cluster_config_path = args.cluster_config
         os.environ["couler_cluster_config"] = cluster_config_path
 
+    os.environ["workflow_name"] = args.workflow_name
```

Comment:
Env variables should be all capitalized, or try not use env to pass parameter?

Reply:
The same action with `cluster_config`, can refactor here in the future.